book/content/software_checklist/01/checklist_matrix.md

```diff
@@ -0,0 +1,107 @@
+# [WIP] Checkmatrix software projects
```

Comment:
I don't know what WIP means. Is that Work In Progress? If so, this should not be in the content.
There is no word 'checkmatrix', maybe use 'checklist matrix' instead?

Reply:
done